springboot/src/main/java/ca/jrvs/apps/trading/AppConfig.java

```diff
@@ -0,0 +1,61 @@
+package ca.jrvs.apps.trading;
+
+import ca.jrvs.apps.trading.dao.MarketDataDao;
+import ca.jrvs.apps.trading.model.config.MarketDataConfig;
+import ca.jrvs.apps.trading.service.QuoteService;
```

Comment:
run a code auto format from you IDE
https://www.notion.so/jarviscanada/IntelliJ-IDEA-Guide-2e124ee013ee4ef3b8e1ee55e3b494aa#d77de86780894612a7d72c479ff67d26

Reply:
@jarviscanada Hello Edward, thank you so much for pointing out all the nonconformance. I have corrected them.